board/questions.py

```diff
@@ -1,6 +1,97 @@
+import logging
+
+
 
 class Questions:
 
     # default constructor
     def __init__(self):
-        self.question = "questions"
+        self.politics = {
+          1: {'question': 'politics quesiton 1', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 100},
+          2: {'question': 'politics quesiton 2', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 200},
+          3: {'question': 'politics quesiton 3', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 400},
+          4: {'question': 'politics quesiton 4', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 800},}
+        self.politics_iterator=1
+
+        self.food = {
+          1: {'question': 'food quesiton 1', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 100},
+          2: {'question': 'food quesiton 2', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 200},
+          3: {'question': 'food quesiton 3', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 400},
+          4: {'question': 'food quesiton 4', 'answers': 'a, b, c, d', 'rAnswer': 'c', 'points': 800},}
+        self.food_iterator=1
+        self.no_category="There are no more questions left in this category. Spin Again"
+
+
+    def check_question(self,catagory):
+
+        print("QUESTIONS:(check_question) -- Checking Catagory: " + str(catagory))
+
+        if catagory == "politics":
```

Comment:
I don't think you should spend time working on this for the Skeletal increment, but I think there is a way to generalize this. Keep track of a dictionary of categories where each category maps to its dictionary of questions. That way you don't have to hard-code the categories.

Reply:
of course, I spent quite of bit of time yesterday trying to generalize this. its a challenge to compare a dictionary value or string to self.CATAGORY dictionary. Since we are passing in strings, we cannot do compare the name of the variables, one being a string and one being a dictionary, as they have totally different values. for now I just wanted to make sure the logic was working and we had a rough idea about the dictionaries we want.